packages/one-trust-notices/src/config/config.ts

```diff
@@ -0,0 +1,44 @@
+import {
+  AWSConfig,
+  FileManagerConfig,
+  LoggerConfig,
+  S3Config,
+} from "pagopa-interop-commons";
+import { z } from "zod";
+import dotenv from "dotenv";
+
+export const OnetrustServiceConfig = FileManagerConfig.and(S3Config)
```

Comment:
S3Config is not needed, no variable named `S3_BUCKET` since here you use dedicated and more specific var names for the buckets.

```suggestion
export const OnetrustServiceConfig = FileManagerConfig
```